### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by carlaKC** [bitcoin-dev-project/sim-ln#123] [General]
If you're interested in contributing to the project and don't know where to start, drop a comment in the discussion and we'll help you find something to work on! 

We also hang out on [discord](https://discord.gg/2QeBWSBs) if you'd like to chat all things sim-ln!

**Comment by richarddushime** [comment]
Hi @carlaKC 
i m interest to contribute to this project and would be happy to get something to work 
as of now i m done setting up the project on my localhost and trying to set the Simulation File Setup

Thank you

**Reply by carlaKC** [reply]
Awesome! Why don't you check out one of these for a first issue: 
* https://github.com/bitcoin-dev-project/sim-ln/issues/136
* https://github.com/bitcoin-dev-project/sim-ln/issues/131
